global.json

```diff
@@ -1,8 +1,8 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview-009764",
+    "dotnet": "3.0.100-preview-010024",
     "vs": {
-      "version": "15.9"
+      "version": "16.0"
     }
   },
   "sdk": {
```

Comment:
@stevenbrix Pretty sure you want the sdk version to match the dotnet version, no?